src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java

```diff
@@ -86,4 +91,23 @@ public double getGlobalConsumption() {
         this.computeConsumption();
         return this.globalConsumption;
     }
+
+    @Override
+    public double getAveragePower() {
+        return this.averagePower;
+    }
+
+    @Override
+    public void setAveragePower(final double averagePower) {
+        this.averagePower = averagePower;
+    }
+
+    @Override
+    public void updateTick(final Duration deltaTime) {
+        final double deltaHours = deltaTime.toSeconds() / 3600;
```

Comment:
Please use `DurationConverter` (and also modify other instances of this in the code you previously wrote, after the PR). Minor, but brings consistency to the codebase. Everything else looks fine, so merge after you fix this